src/utils.js

```diff
@@ -32,7 +32,7 @@ async function call (host, path, data, agent) {
       throw e
     }
   }
-  if (body?.error !== undefined) throw new Error(body?.errorMessage)
```

Comment:
Is .error a string ?

Reply:
Yes, errors on the API are - from what I've seen at least - always strings on the object. Sometimes it provides a user friendly response, which errorMessage handles, and sometimes it doesn't, for example with a ForbiddenOperationException.

![](https://media.discordapp.net/attachments/957177813155344425/957579028049592360/ForestgreenSunbear.png)